### ISSUE TITLE
Orientation and reconstruction quality 

### ISSUE THREAD

**Issue by kohlisimranjit** [czq142857/IM-NET-pytorch#3]
Hi,
Thank you for the good work and providing the data and trained models. It helps a lot! tried to test the reconstruction on our dataset with the provided pre-trained model and we get poor results. 
![image](https://user-images.githubusercontent.com/5343099/98442976-5d93f580-20d6-11eb-9795-3578a2f01fd9.png)
The left most part is our voxelization of the ground truth mesh (shown on the center). The right most part is the reconstruction.
We have not trained or fine tuned on our dataset. I was wondering what could the potential issue be. Could it be orientation or something else?

**Comment by czq142857** [comment]
Yes. Please make sure the orientation of your data is consistent with our provided data.
The 3D CNN encoder is not invariant to rotation and scale. So you also need to normalize your data correctly before doing voxelization. Please normalize each shape so that the diagonal of its bounding box equals to one. (If you use binvox you are likely to place the shape in a unit cube so that the longest edge in its bounding box is one.)

Best,
Zhiqin

**Comment by kohlisimranjit** [comment]
Thanks a ton! And just to confirm so if I have my own mesh I need to run the following command for each file to get the 64*64*64 voxel:
https://github.com/czq142857/IM-NET/issues/10#issuecomment-650843347

**Comment by czq142857** [comment]
Yes.
binvox -bb -0.5 -0.5 -0.5 0.5 0.5 0.5 -d 64 -e input.obj
"-bb" defines the bounding box.

**Comment by kohlisimranjit** [comment]
thanks!
